src/transactions/sendBtcTransaction.ts

```diff
@@ -26,20 +27,23 @@ export const sendBtcTransaction = async (options: SendBtcTransactionOptions) =>
   const provider = await getProvider();
 
   if (!provider) {
-    throw new Error('No Bitcoin Wallet installed');
+    throw new Error('No Bitcoin wallet installed');
   }
-  if (!recipients) {
-    throw new Error('Recipient is required');
+  
+  const { recipients, senderAddress } = options.payload;
```

Comment:
This line seems to be repeated. See line 25

Reply:
Fixed in b5e8123d1408a178cfa847c80d1c160ae316c53b.

Reply:
Oops, that was a merging blunder on my part 🙈